democracy/migrations/0009_section_type_model.py

```diff
@@ -7,12 +7,48 @@
 from django.db import migrations, models
 import django.db.models.deletion
 import django.utils.timezone
-from democracy.models.initial_data import create_initial_section_types
+
+
+class InitialSectionType:
```

Comment:
There are later migrations that assume there exists a type `introduction`. While that should not matter if there is no actual data in the db, I still think this is more safer way without any drawbacks.